<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>511</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>单词消除游戏</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="unameText">
      <property name="baseSize">
       <size>
        <width>5</width>
        <height>51</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密    码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="pwdText">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>540</y>
     <width>641</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="loginButton">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="registrationButton">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>431</y>
     <width>131</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>类    型：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeCbx">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>511</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>出题者</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>玩家</string>
    </property>
   </item>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>typeCbx</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
